torch/testing/_internal/common_nn.py

```diff
@@ -4881,106 +4948,90 @@ def _do_test(self, test_case, module, input):
             output_ip.backward(grad)
             test_case.assertEqual(input.grad, input_ip.grad)
 
-        if isinstance(input, torch.LongTensor) and TEST_CUDA:
+        def test_module_parameters(dtype, device_id=None):
```

Comment:
nit: `assert_module_parameters_are(tensor_type, device_id)` gets the idea across a bit better.

Also, the tensor type is not actually a "dtype" so we shouldn't call the argument "dtype" -- dtypes are things like `torch.float`, `torch.double`.

Reply:
sure